### PR TITLE
Feature/httpfs callback

### PR DESCRIPTION
**Pull request by Nwitz**

### CODE REVIEW

File: src/Httpfs/Httpfs.java [hunk 1]
```diff
@@ -15,6 +16,8 @@
         version = "1.0")
 public class Httpfs implements Runnable {
     private int _port = 8080;
+    // TODO: remove
+    private FileManager fileManager = new FileManager(directory = Paths.get("").toAbsolutePath());
```
Comment (Swess) [inline src/Httpfs/Httpfs.java]: Is it still to remove ??

File: src/Httpfs/FileManager.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+
+    FileManager(Path directory) {
+        this.directory = directory;
+    }
+
+    /**
+     * Ensure path is within directory
+     */
+    public boolean isFilePathValid(String path){
```
Comment (Swess) [inline src/Httpfs/FileManager.java:19]: This seems good. But it's never called from writeToFile or readFile.

Comment (Nwitz) [reply src/Httpfs/FileManager.java:19]: I see the check for /../ when parsing the url path, so any url with that won't even make it to this point. Left it in incase this changes.

File: src/Httpfs/Httpfs.java (hunk 1 above)
Comment (Nwitz) [reply src/Httpfs/Httpfs.java]: removed - No need to create instance here with meaningless directory

**Review by Swess — APPROVED** [review]

**Pull request merged**